src/psyclone/psyir/tools/dependency_tools.py

```diff
@@ -650,26 +640,26 @@ def _array_access_parallelisable(self, loop_variables, var_info):
                         self._add_message(LazyString(
                             lambda node=write_access.node:
                                 (f"The write access to '"
-                                 f"{self._language_writer(node)}' causes "
+                                 f"{node.debug_string()}' causes "
```

Comment:
Good idea. I have another PR somewhere where I've messed with this but my solution is not as good as yours (I forgot about `debug_string()`).

Reply:
pylint complains that FortranWriter is imported but unused.